### DISCUSSION TITLE
UST Stuck in the wormhole. Ethereum to Solana

### DISCUSSION THREAD

**Discussion by radusiek16** [wormhole-foundation/wormhole#243] [Problems / bugs] [answered]
Hey, my funds are stuck in a wormhole. It says "waiting for solana completion".
TX id: 0x466c1367785cc41eb3208be5c29d0fe1848142b64bf4e4e0734d901e3d16bcf0
Solana wallet: 7ggX7iVmFL2UXtwbLbpZt3RSGyxsz8XY4Aer6MdqVQs9
Thank you!

**Comment by hendrikhofstadt** [comment, marked as answer]
Confirmed in https://explorer.solana.com/tx/5PgnyhKBVshL9TRwSVuQTGxiK1FWVgqUgz6ddpLR53QkZFn9KDS9vZZw7RtcauHUCKp2ehTro7VmWidU9Gat1eB4
